Makefile

```diff
@@ -74,7 +74,7 @@ test-architecture: ## check architecture constraints between packages
 	go test ./pkg/architests/... -v
 
 test-client: ## runs test that checks sdk.Client without instrumentedsql
-	SF_TF_NO_INSTRUMENTED_SQL=1 SF_TF_GOSNOWFLAKE_LOG_LEVEL=debug go test ./pkg/sdk/internal/client/... -v
+	SF_TF_NO_INSTRUMENTED_SQL=1 go test ./pkg/sdk/internal/client/... -v
```

Comment:
do we have a follow-up task to turn on logging back here?

Reply:
I thought this is not needed anymore. I can use `SNOWFLAKE_DRIVER_TRACING` here instead in the next PR.

Reply:
The problem here is that client initialization does not use environment variables (because using them from the Terraform level caused order problems), but the old SF_TF_GOSNOWFLAKE_LOG_LEVEL was handled additionally. We could handle the new env the same way as the old one, this could be the only env that is currently directly supported by the client initialization and the ordering is good: envs > toml.